.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Why did you decide to disable `package-lock`?

Reply:
Lockfiles make sense for applications but not for libraries, in my experience.